content/docs/hooks-overview.md

```diff
@@ -1,24 +1,24 @@
 ---
 id: hooks-overview
-title: Hooks at a Glance
+title: एक नज़र में Hooks
 permalink: docs/hooks-overview.html
 next: hooks-state.html
 prev: hooks-intro.html
 ---
 
-*Hooks* are a new addition in React 16.8. They let you use state and other React features without writing a class.
+React 16.8 में *Hooks* एक नया फीचर ऐड हुआ है। वे आपको class लिखे बिना state और अन्य React सुविधाओं का उपयोग करने देता हैं।
 
-Hooks are [backwards-compatible](/docs/hooks-intro.html#no-breaking-changes). This page provides an overview of Hooks for experienced React users. This is a fast-paced overview. If you get confused, look for a yellow box like this:
+Hooks [बैकवर्ड-कम्पेटिबल](/docs/hooks-intro.html#no-breaking-changes) है। यह पेज अनुभवी React उपयोगकर्ताओं के लिए Hooks का ओवरव्यू प्रदान करता है। यह एक तेज ओवरव्यू है। यदि आप उलझन में पड़ जाते हैं, तो इस तरह पीले बॉक्स की तलाश करे:
```

Comment:
बैकवर्डs

Reply:
है -> हैं

Reply:
तेज ओवरव्यू -> could be replaced by something better?

Reply:
तो इस तरह पीले बॉक्स की तलाश करे: -> check grammar

Reply:
> तेज ओवरव्यू -> could be replaced by something better?

@saranshkataria 
I am not able to think of something meaningful other than this. Do you have something in mind?
Also, I have pushed the other changes.

Reply:
तेज गति ka overview?

Reply:
makes sense.